### DISCUSSION TITLE
[Suggestion] Lists w/ stars on private repos only should be hidden from public view

### DISCUSSION THREAD

**Discussion by adamlui** [community/community#44154] [Other Feature Feedback, Questions, & Ideas]
When adding private repos to lists for personal organization for example, they appear under Stars > Lists to the public, even if the public lengths are 0. Since this is not useful for public viewing, they should be hidden instead if all elements are private

**Comment by TomaszZel** [comment]
Ok

**Reply by adamlui** [reply]
Ok indeed!

**Comment by Sofa1113** [comment]
> Ok
